### PR TITLE
OSDOCS-10809 Ingress Capability Warning Release Note

### PR DESCRIPTION
**Pull request by jmanthei**
Version(s): 4.16

Issue: https://issues.redhat.com/browse/OSDOCS-10806

Link to docs preview: https://77675--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-installation-and-update-ingress-capability_release-notes

QE review:
- [x] QE has approved this change: https://github.com/openshift/openshift-docs/pull/77601

Additional information:

### CODE REVIEW

**Comment by tmalove** [comment]
/remove-label peer-review-in-progress
/label peer-review-done

Preview link: https://77675--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-installation-and-update-ingress-capability_release-notes

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -74,6 +74,16 @@ In {product-title} {product-version}, the installation program uses Cluster API
 
 Additionally, SSH access to control plane and compute machines is no longer open to the machine network, but is restricted to the security groups associated with the control plane and compute plane machines.
 
+[id="ocp-4-16-installation-and-update-ingress-capability_{context}"]
+==== Ingress capability
+
+Ingress capability is now a configurable cluster capability and is optional for Red{nbsp}Hat Hypershift. It is not configurable and always enabled for standalone {product-title}.
```
Comment (mburke5678) [inline release_notes/ocp-4-16-release-notes.adoc:80]:
```suggestion
Ingress capability is now a configurable cluster capability and is optional for Red{nbsp}Hat Hypershift. It is not configurable and is always enabled for standalone {product-title}.
```

Comment (mburke5678) [reply release_notes/ocp-4-16-release-notes.adoc:80]: @jmanthei   I think we need an _is_ here, or am I being too picky?

Comment (jmanthei) [reply release_notes/ocp-4-16-release-notes.adoc:80]: It's correct either way, but I'm happy to add an "is" if you think it's more clear!

**Comment by jmanthei** [comment]
@mburke5678 added the "is"

**Comment by jmanthei** [comment]
Please resume merge? I resolved the merge conflict.

/label merge-review-needed.

**Review by skopacz1 — APPROVED** [review]
Merge review LGTM

**Pull request merged**